docs/ref/models/querysets.txt

```diff
@@ -1887,6 +1887,45 @@ See the :doc:`/topics/db/sql` for more information.
   filtering. As such, it should generally be called from the ``Manager`` or
   from a fresh ``QuerySet`` instance.
 
+``comment()``
+~~~~~~~~~~~~~
+
+.. versionadded:: 4.2
+
+.. method:: comment(message)
+
+Adds an SQL comment to be inserted into the resultant SQL statement, after
+``SELECT``/``UPDATE``, and if applicable, ``DISTINCT``.. For example::
+
+    Entry.objects.comment("blog/views.py:50").all()
```

Comment:
I'm not sure if references to the lines of code are the best examples :thinking:

Reply:
Maybe not, my use case would be something like `comment("trace_id:12345")` or `comment("request_id:3c86a5ec-8b87-4304-9a9b-b42f49787fad")` or maybe `comment(f"user_id:{request.user.id}")`.

Reply:
Tracing which view generated a query is useful IMO, but maybe not the specific line of code though, since it could change between deployments.